lib/admin.php

```diff
@@ -381,7 +381,7 @@ function wc_admin_output_breadcrumb( $section ) {
  *
  * @todo Icon Placeholders for the ActivityPanel, when we implement the new designs.
  */
-function woocommerce_embed_page_header() {
+function wc_admin_woocommerce_embed_page_header() {
```

Comment:
We can remove the `woocommerce` portion of the prefix, so the name will be `wc_admin_embed_page_header`.

Reply:
Hi Paul,

Thank you for your thoughts, I have updated my PR with the changes. Please review it and let me know your feedback on the same.

Thanks,
Ronak